[R2] get started guide (#3445)
cloudflare/cloudflare-docs: products/r2/src/guide.md

```sh
$ wrangler login
Allow Wrangler to open a page in your browser? [y/n]
y
💁  Opened a link in your default browser: https://dash.cloudflare.com/oauth2/...
```

Open the browser, log into your account, and select **Allow**. This will send an OAuth Token to Wrangler so it can deploy your scripts to Cloudflare.

## 3. Create your bucket

Create your bucket by running `wrangler r2 bucket create <your-bucket-name>`:

```sh
wrangler r2 bucket create <your-bucket-name>
```

To check that your bucket was created, run:

```sh
wrangler r2 bucket list
```

## 4. Bind your bucket to a Worker

You will need to bind your bucket to a Worker. Run the [`wrangler generate`](/workers/cli-wrangler/commands#generate) command to create a Worker using a [template](/workers/get-started/quickstarts#templates). Wrangler templates are git repositories that are designed to be a starting point for building a new Cloudflare Workers project. By default, the [default starter](https://github.com/cloudflare/worker-template) template will be used to generate your new Worker project:

```sh

> wrangler generate r2-demo https://github.com/cloudflare/worker-template

Author: ```suggestion
wrangler generate r2-demo
```

```

Next, find your newly generated `wrangler.toml` file in your project's directory and update `account_id` with your Cloudflare Account ID. 

Find your Account ID by going logging in to the Cloudflare dashboard > **Overview** > move down to **API** > and select **Click to copy** to copy your **Account ID**. Or run the `wrangler whoami` command [to copy your Account ID](/workers/get-started/guide#6-preview-your-project).

```sh
name = "r2-demo"
type = "javascript"
compatibility_date = "2022-02-10"
 
account_id = "your-account-id" # ← Replace with your Account ID.
workers_dev = true
```

> To bind your R2 bucket to your Worker, add the following to your `wrangler.toml` file. You can customize `binding` and `bucket_name` to suit your project:

Author: ```suggestion
To bind your R2 bucket to your Worker, add the following to your `wrangler.toml` file. Update `binding` to a valid JavaScript identifier and `bucket_name` to the `<your-bucket-name>` you used to create your bucket in step 3:
```

```sh
[[r2_buckets]]

> binding = 'MY_BUCKET'
> bucket_name = 'my-bucket'

Author: ```suggestion
binding = '<your_bucket_name>'
bucket_name = '<your_bucket_name>'
```

```

Find more detailed information on configuring your Worker in the [Wrangler Configuration documentation](/workers/cli-wrangler/configuration).

<!--
// Buckets API 
// User downloads a file.
-->

> ## 5. Buckets API

Author: ```suggestion
## 5. Accessing your R2 bucket from your Worker
```
